svelte/components/Nft/NftMint.svelte

```diff
@@ -318,22 +321,30 @@
                     <div>
                       <span class="kre-market-info-title label-big kre-no-wrap-title">Mint price</span>
                       <span class="kre-market-info-value label-big kre-no-wrap-title"
-                        >{utils.formatEther(collection?.price)} (Eth)</span
+                        >{utils.formatEther(collection?.price)} ({getNetwork(chainId).nativeCurrency.symbol})</span
                       >
                     </div>
                   {/if}
                   {#if collection?.royaltyFee?.gt(0)}
                     <div>
                       <span class="kre-market-info-title label-big">royalties</span>
-                      <span class="kre-market-info-value label-big">{collection?.royaltyFee.div(100)} %</span>
+                      <span class="kre-market-info-value label-big">{collection?.royaltyFee.toNumber() / 100} %</span>
                     </div>
                   {/if}
                   {#if collection?.royaltyAccount !== constants.AddressZero}
                     <div>
                       <span class="kre-market-info-title label-big kre-no-wrap-title">royalty receiver</span>
-                      <span class="kre-market-info-value label-big overflow-ellipsis">{collection?.royaltyAccount}</span>
+                      <span class="kre-market-info-value label-big overflow-ellipsis"
+                        ><a href={explorerAddressUrl(chainId, collection?.royaltyAccount)} class="link"
+                          >{collection?.royaltyAccount}</a
+                        ></span
+                      >
                     </div>
                   {/if}
+                  <div>
+                    <span class="kre-market-info-title label-big kre-no-wrap-title">fees</span>
+                    <span class="kre-market-info-value label-big overflow-ellipsis">{config.treasury.fee / 100} %</span>
```

Comment:
use BigNumber (or Number ok for fees %)

Reply:
config.treasury.fee is Number ?

Reply:
yes read value comes from json, so Number